best_practices/templates.rst

```diff
@@ -51,6 +51,10 @@ Another advantage is that centralizing your templates simplifies the work
 of your designers. They don't need to look for templates in lots of directories
 scattered through lots of bundles.
 
+.. best-practice::
+
+    Use lowercased snake_case for directory and template names.
```

Comment:
In this moment, my thinking is that the *old* way seems the best, as it's very simple: everything just matches: no transformations (of course, you can do whatever you want).